src/test/mir-opt/simplify_match/rustc.main.SimplifyBranches-after-copy-prop.diff

```diff
@@ -10,19 +10,24 @@
       }
   
       bb0: {
-          nop;                             // bb0[0]: scope 0 at $DIR/simplify_match.rs:6:11: 6:31
-          nop;                             // bb0[1]: scope 0 at $DIR/simplify_match.rs:6:17: 6:18
-          nop;                             // bb0[2]: scope 0 at $DIR/simplify_match.rs:6:21: 6:26
-          nop;                             // bb0[3]: scope 1 at $DIR/simplify_match.rs:6:28: 6:29
-          nop;                             // bb0[4]: scope 0 at $DIR/simplify_match.rs:6:30: 6:31
--         switchInt(const false) -> [false: bb1, otherwise: bb2]; // bb0[5]: scope 0 at $DIR/simplify_match.rs:7:9: 7:13
--                                          // ty::Const
--                                          // + ty: bool
--                                          // + val: Value(Scalar(0x00))
--                                          // mir::Constant
--                                          // + span: $DIR/simplify_match.rs:6:21: 6:26
--                                          // + literal: Const { ty: bool, val: Value(Scalar(0x00)) }
-+         goto -> bb1;                     // bb0[5]: scope 0 at $DIR/simplify_match.rs:7:9: 7:13
```

Comment:
Since `CopyProp` doesn't actually do anything here, please change it so that we see the const prop diff here

Reply:
I just did. Should I also move it to the `const_prop` folder?